pkg/istio/envoy_filters.go

```diff
@@ -1,4 +1,4 @@
-package common
```

Comment:
Moving package to have package `github.com/kuadrant/kuadrant-controller/pkg/common` as a dependency for package `github.com/kuadrant/kuadrant-controller/apis/apim/v1alpha1` (not the other way around as it was until now)